notebooks/official/sdk/SDK_AutoML_Video_Classification.ipynb

```diff
@@ -60,7 +63,7 @@
       "source": [
         "## Overview\n",
         "\n",
-        "This notebook demonstrates how to create an AutoML Video Classification Model, with a Vertex AI video dataset, and how to serve the model for batch prediction. It requires you provide a bucket where the dataset will be stored.\n",
+        "This notebook demonstrates how to create an AutoML Video Classification Model, with a Vertex AI video dataset, and how to serve the model for batch prediction. It requires you to provide a bucket where the dataset is stored.\n",
```

Comment:
Change "AutoML Video Classification Model" (only "AutoML" is a product name):

- AutoML video classification model